mediapipe/calculators/geti/inference/BUILD

```diff
@@ -41,21 +41,27 @@ cc_library(
         "classification_calculator.cc",
         "detection_calculator.cc",
         "instance_segmentation_calculator.cc",
+        "keypoint_detection_calculator.cc",
         "openvino_inference_adapter_calculator.cc",
         "rotated_detection_calculator.cc",
         "segmentation_calculator.cc",
         "model_infer_request_image_calculator.cc",
+        "model_infer_http_request_calculator.cc",
     ],
     hdrs = [
         "anomaly_calculator.h",
         "classification_calculator.h",
         "detection_calculator.h",
         "instance_segmentation_calculator.h",
         "kserve.h",
+        "keypoint_detection_calculator.h",
         "openvino_inference_adapter_calculator.h",
         "rotated_detection_calculator.h",
         "segmentation_calculator.h",
         "model_infer_request_image_calculator.h",
+        "model_infer_http_request_calculator.h",
+        "http_payload.h",
```

Comment:
Instead depend on OVMS libhttpclientconnection & httppayload.

Reply:
This would add an ovms dependency to an existing mediapipe repository, we want to keep the repos the same for easier testing.

Reply:
There is dependency but it is hidden from buildsystem. Even if you add dependency to those two it doesnt mean that you depend on (and link) with whole OVMS.

You may need to first make those two visible from outside of OVMS (visibility public).

Reply:
This should make that change easier:
https://github.com/openvinotoolkit/model_server/pull/2790/files